hugo/content/guides/builtin-library.md

```diff
@@ -53,6 +54,8 @@ As a last step, we have to bind our newly created workspace manager:
 
 ```ts
 // Add this to the `hello-world-module.ts` included in the yeoman generated project
+export type HelloWorldSharedServices = LangiumSharedServices;
+
 export const HelloWorldSharedModule: Module<HelloWorldSharedServices, DeepPartial<LangiumSharedServices>> = {
```

Comment:
Suggestion: The first and second type arguments shouldn't deviate here. In case the dev changes `HelloWorldSharedService`, the following is better:
```suggestion
export const HelloWorldSharedModule: Module<HelloWorldSharedServices, DeepPartial<HelloWorldSharedServices>> = {
```

Reply:
Good point, thanks!